Letterbook.IntegrationTests/Fixtures/HostFixture.cs

```diff
@@ -192,6 +221,14 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
 				{
 					options.ForwardAuthenticate = "Test";
 				});
+				services.AddOpenTelemetry()
+					.WithTracing(tracer =>
+					{
+						tracer.AddInMemoryExporter(_spans);
```

Comment:
Here, we configure tracing to export to an in-memory collection. This is actually in addition to the regular config, so if you have tempo running during your integration tests, you can also view those spans in grafana.